src/basic/capability-util.c

```diff
@@ -138,6 +138,42 @@ int capability_ambient_set_apply(uint64_t set, bool also_inherit) {
         return 0;
 }
 
+int capability_ambient_set_clear(uint64_t set) {
```

Comment:
hmm, iiuc the parameter specifies the ones to keep, right? hence call it "keep" or so, to make this clearer?